build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2004,6 +2013,8 @@ message SpliceBlobRequest {
   //  2. It allows servers with different storage backends to dispatch the
   //     request to the correct storage backend based on the size and/or the
   //     hash of the blob.
+  //  3. It allows the server to check if existing chunking information is 
+  //     available for the blob.
```

Comment:
This is essentially a rewording of point (1), right?

It's worth keeping in mind that this field is still optional and not required, which makes it a SHOULD, not a MUST. If the client is CPU-constrained and prefers to skip hashing the giant file, it should be able to send all the chunks to the server and trust the digest the server returns.

If you are interested in implementing a server that enforces that this field is mandatory, we can try creating a new capability field, telling clients that they MUST set this field or expect an error otherwise.

Reply:
See https://github.com/bazelbuild/remote-apis/pull/356. I think starting stricter is better, and this should be required. Its still an unreleased API